### ISSUE TITLE
createRef and forwardRef

### ISSUE THREAD

**Issue by rosenfeld** [thysultan/dio.js#63]
Hi, the createRef and forwardRef helpers seem quite interesting (specially createRef). Do you have any plans to add them to Dio.js?

https://reactjs.org/docs/refs-and-the-dom.html
https://reactjs.org/docs/forwarding-refs.html

**Comment by thysultan** [comment]
Yes, the semantics behind `createRef` are kind of already supported minus the top-level API to create the object.

**Comment by rosenfeld** [comment]
Thanks! I'm having some issues with debugging errors with Dio.js. I'll open a new ticket for that.
